pdns/recursordist/rec_channel_rec.cc

```diff
@@ -2110,7 +2110,7 @@ RecursorControlChannel::Answer RecursorControlParser::getAnswer(int s, const str
             "set-carbon-server                set a carbon server for telemetry\n"
             "set-dnssec-log-bogus SETTING     enable (SETTING=yes) or disable (SETTING=no) logging of DNSSEC validation failures\n"
             "set-event-trace-enabled SETTING  set logging of event trace messages, 0 = disabled, 1 = protobuf, 2 = log file, 3 = both\n"
-            "trace-regex [regex]              emit resolution trace for matching queries (empty regex to clear trace)\n"
+            "trace-regex [regex file]         emit resolution trace for matching queries (no arguments clears tracing)\n"
```

Comment:
this does not make clear that passing just the regex is still allowed

Reply:
just theregex is not allowed.... Ether nothing to clear the regex, or two args: a regex and a file, which may be `-`.

Reply:
ah! so a backwards-incompatible change. Fine, but should be very clear then :)